clang/test/CXX/drs/cwg29xx.cpp

```diff
@@ -6,7 +6,7 @@
 // RUN: %clang_cc1 -std=c++23 -pedantic-errors -verify=expected %s
 // RUN: %clang_cc1 -std=c++2c -pedantic-errors -verify=expected %s
 
-namespace cwg2917 { // cwg2917: 20 open 2024-07-30
+namespace cwg2917 { // cwg2917: 20 review 2024-07-30
```

Comment:
@Sirraide you can update the test separately, or push the update directly into this PR. Or I can merge this as is if you don't have time to get to this during the week. We have a date reference to what we implement, so it's not like we'll be lying to users.

Reply:
> you can update the test separately

Well the issue is I’m not entirely sure how we should handle this case: The suggested resolution is different from the proposed resolution (we currently implement the former), and @cor3ntin said it would be better to wait and not make a change to the implementation for now (https://github.com/llvm/llvm-project/pull/101448#issuecomment-2307838729), so my understanding is that saying that we implement the 2024-07-30 resolution would be the most accurate?

Reply:
OK, let's just update the status.